src/cloud/docker/docker-mftf.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Magento Functional Testing
+group: cloud-guide
+functional_areas:
+  - Cloud
+---
+
+Magento CLoud Docker supports Functional Testing with [Magento Functional Testing Framework]({{site.baseurl}}/mftf/docs/introduction.html)
+
+This means that you're able to run Functional Tests locally and target them on your Cloud Docker or remote instance.
+
+{:.bs-callout-info}
+To be able to run MFTF use [Magento Cloud Docker][cloud-docker-repo] version 1.0 or higher
+
+1. Prepare environment
+
+    - Add MFTF dependency via Composer tou your project,
+
+    ```bash
+    composer require "magento/magento2-functional-testing-framework" --no-update
+   ```
+
+   - Install new Composer Dependency
+   
+   ```bash
+    composer update
+    ```
+
+1. Generate `docker-compose.yml` file
+
+    ```bash
+    ./vendor/bin/ece-docker build:compose --with-selenium --no-cron
+    ```
+
+1.  Start Cloud Docker
+
+    Optionally you can setup Cloud Docker to work in [Developer Mode]({{site.baseurl}}/cloud/docker/docker-mode-developer.html)
+    
+    ```bash
+    ./bin/magento-docker up
+    ./bin/magento-docker ece-redeploy
+    ```
+
+1. Prepare Magento
+
+    - Add environment variables specific to MFTF
+    
+    {:.bs-callout-info}
+    You may use remote URL in `MAGENTO_BASE_URL` variable below
+    
+    ```bash
+    CONFIG="MAGENTO_BASE_URL=http://magento2.docker/
+    MAGENTO_BACKEND_NAME=admin
+    MAGENTO_ADMIN_USERNAME=admin
+    MAGENTO_ADMIN_PASSWORD=123123q
+    MODULE_WHITELIST=Magento_Framework,Magento_ConfigurableProductWishlist,Magento_ConfigurableProductCatalogSearch
+    SELENIUM_HOST=selenium"
+   
```

Comment:
Just checking:  Is the quote syntax correct here:  open on the CONFIG value; close after SELENIUM_HOST value?

Reply:
Yes